### ISSUE TITLE
Nommage des tables de la bdd

### ISSUE THREAD

**Issue by BenjaminPiscart** [MaVoix/candidature#53]
Afin de rester cohérent de bout en bout (l'usage de l'anglais), je vous suggère de renommer la table "candidature" par "application".

**Comment by ColinMaudry** [comment]
Effectivement, nous aurions dû être plus rigoureux dans le choix des noms de tables. Pour ma part je suis occupé par les tests des dernières features, et sur le lancement du développement de la plateforme de collecte de dons.

Si vous souhaitez faire une Pull Request, vous êtes le bienvenu :)
